build/buildBuildImages.sh

```diff
@@ -336,7 +336,7 @@ function buildVsoFocalImage() {
 
 	BuildAndTagStage "$BUILD_IMAGES_VSO_FOCAL_DOCKERFILE" intermediate
 	echo
-	echo "-------------Creating VSO Ubuntu <focal> build image-------------------"
+	echo "-------------Creating VSO full build image-------------------"
```

Comment:
full or focal? let's use a standard naming convention. Full makes me think it has all the SDKs, focal means it has some?